jp/lessons/basics/enum.md

```diff
@@ -101,38 +111,56 @@ iex> Enum.map([0, 1, 2, 3], fn(x) -> x - 1 end)
 
 ### min
 
-コレクションの中で最小の(`min`)値を探します:
+コレクションの中で最小の(`min/1`)値を探します:
 
 ```elixir
 iex> Enum.min([5, 3, 0, -1])
 -1
 ```
 
+`min/2`も同様ですが、コレクションが空である場合、最小値を生成するための関数を渡します:
+
+```elixir
+iex> Enum.min([], fn -> :foo end)
+:foo
+```
+
 ### max
 
-コレクションの中で最大の(`max`)値を返します:
+コレクションの中で最大の(`max/1`)値を返します:
 
 ```elixir
 iex> Enum.max([5, 3, 0, -1])
 5
 ```
 
+`max/2`と`max/1`の関係は`min/2`と`min/1`の関係と同じです:
+
+```elixir
+Enum.max([], fn -> :bar end)
+:bar
+```
+
 ### reduce
 
-`reduce`を用いることで、コレクションをまとめ、そこから単一の値を抽出することができます。この処理を実行するにはオプションとしてアキュムレータ(積算器。この例では`10`)を関数に渡しますが、アキュムレータが与えられない場合にはコレクションの最初の値が用いられます:
+`reduce/3`を用いることで、コレクションをまとめ、そこから単一の値を抽出することができます。この処理を実行するにはオプションとしてアキュムレータ(積算器。この例では`10`)を関数に渡しますが、アキュムレータが与えられない場合にはコレクションの最初の値が用いられます:
 
 ```elixir
 iex> Enum.reduce([1, 2, 3], 10, fn(x, acc) -> x + acc end)
 16
+
 iex> Enum.reduce([1, 2, 3], fn(x, acc) -> x + acc end)
 6
+
 iex> Enum.reduce(["a","b","c"], "1", fn(x,acc)-> x <> acc end)
 "cba1"
 ```
 
 ### sort
 
-コレクションをソートするのは1つ、ではなく、2つある`sort`関数を使えば簡単です。1つ目の選択として用意されているのはElixirのterm orderingを使ってソート順序を決めるというものです:
+コレクションをソートするのは1つ、ではなく、2つあるソート関数を使えば簡単です。
```

Comment:
two 、?

Reply:
@riseshia do you wish to address this or should I merge?

Reply:
fixed 4b98122

Reply:
I'm sorry to be late ;)